src/ffi.rs

```diff
@@ -1040,7 +1040,7 @@ macro_rules! extern_c_block {
 }
 
 #[cfg(target_os = "windows")]
-extern_c_block!("libhwloc");
+extern_c_block!("hwloc");
```

Comment:
Given that the Windows CI currently works with link_name = "libhwloc", I suspect this will break it. Is there a way we can hint the custom build into having the same link_name as the official hwloc builds ?